<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Police Killings Analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>General</string>
     </attribute>
     <widget class="QPushButton" name="button1">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Stats</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Deaths graph for the year :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Stats</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Racial distribution of month-wise killings</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Stats</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Race-wise distribution of the killings</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>230</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Stats</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>age wise distribution of the killings</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Gender wise distribution of the killings</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>State-wise distribution of the killings</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button5">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Stats</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Stats</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button7">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Stats</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Mode of fleeing</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button8">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>310</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Stats</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Signs of Mental illness</string>
      </property>
     </widget>
     <widget class="QComboBox" name="yearSelector1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Statistical Tools</string>
     </attribute>
     <widget class="QLabel" name="label_17">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Select the desired data to find:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Min :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Age</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="ViewStatsBtn">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>280</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>View values</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>41</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Max :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Mean :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Age</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Age</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Age</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>120</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>150</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>180</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>210</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Median:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>240</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Standard Deviaton :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>240</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Age</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>240</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>T-test</string>
     </attribute>
     <widget class="QPushButton" name="TtestBtn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>230</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Perform T-test</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Avenir</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select values to perform T-test Hypothesis testing:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboYear">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>2015</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2016</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Year :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Month :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboMonth">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>January</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>February</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>March</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>April</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>May</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>June</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>July</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>August</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>September</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>October</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>November</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>December</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Year :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboMonth2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>160</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>January</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>February</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>March</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>April</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>May</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>June</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>July</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>August</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>September</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>October</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>November</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>December</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Month :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboYear2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>2015</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2016</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Avenir</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Note: Hypothessis 0 is always the null situation.</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="textEdit1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_70">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Avenir</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select states to compare:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>ANOVA</string>
     </attribute>
     <widget class="QLabel" name="label_64">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Avenir</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select values to perform ANOVA Hypothesis testing:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_65">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Avenir</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Note: Hypothessis 0 is always the null situation.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="anovabtn">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>240</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Perform ANOVA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_66">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>State:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="tEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_68">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>State:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="state1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Arizona</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>California</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Texas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Florida</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ohio</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_69">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>State:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="state2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>California</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Arizona</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Texas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Florida</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ohio</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="state3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>160</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Texas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Arizona</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>California</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Florida</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ohio</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_67">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Avenir</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select states to compare how different are the samples of age groups</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_71">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Avenir</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>on the basis of Analysis of Variance:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
